Recrutify/Recrutify.Services/Validators/CreateCandidateValidator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Linq;
+using FluentValidation;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Validators
+{
+    public class CreateCandidateValidator : AbstractValidator<CandidateCreateDTO>
+    {
+        public CreateCandidateValidator()
+        {
+            RuleFor(c => c.Name)
+                .NotNull()
+                .NotEmpty()
+                .MaximumLength(50);
+            RuleFor(c => c.Surname)
+                .NotNull()
+                .NotEmpty()
+                .MaximumLength(50);
+            RuleFor(c => c.PhoneNumber)
+                .NotNull()
+                .NotEmpty()
+                .Matches(@"^\+?(\d[\d\- ]+)?(\(?[\d\- ]+\))?([\d\- ])+\d$");
+            RuleFor(c => c.Email)
+                .NotNull()
+                .NotEmpty()
+                .EmailAddress();
+            RuleFor(c => c.Contacts)
+                .NotNull()
+                .NotEmpty();
+            RuleFor(t => t.Contacts.ToList()).
+                Must(t => t.Any(list => list.Type == "Skype"))
```

Comment:
Rename t to c, list to contact - it's not list it's single object. Move Skype to const string field of validator class